<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>planner</class>
 <widget class="QMainWindow" name="planner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1127</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>planner</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frame_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>20</y>
         <width>261</width>
         <height>331</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="addPlan">
            <property name="styleSheet">
             <string notr="true">#addPlan{
border:1px solid transparent;
}</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Minimum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>78</width>
              <height>126</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="planerr">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Time:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTimeEdit" name="pt"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_3">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Date:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="pd"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="dataSend">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">#dataSend{
max-height:40px;


}</string>
          </property>
          <property name="text">
           <string>Send Data</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QListView" name="listView">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>30</y>
         <width>291</width>
         <height>391</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>430</y>
         <width>75</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Show Plans</string>
       </property>
      </widget>
      <widget class="QPushButton" name="budgetbut">
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>600</y>
         <width>75</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Budget</string>
       </property>
      </widget>
      <widget class="QPushButton" name="fitnessbut">
       <property name="geometry">
        <rect>
         <x>720</x>
         <y>600</y>
         <width>75</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Fitness</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="editingPlan">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>480</y>
         <width>271</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="updateBut">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>610</y>
         <width>75</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
      <widget class="QPushButton" name="deleteBut">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>610</y>
         <width>75</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>790</x>
         <y>30</y>
         <width>291</width>
         <height>391</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="plan1">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="plan3">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="plan2">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="plan4">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QDateEdit" name="datecome">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>580</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QTimeEdit" name="timecome">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>580</y>
         <width>118</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>440</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
